flowzone.yml

```diff
@@ -1398,7 +1422,38 @@ jobs:
           echo "branch_tag=${branch_tag}" >> $GITHUB_OUTPUT
           echo "sha_tag=${sha_tag}" >> $GITHUB_OUTPUT
           echo "version_tag=${version_tag}" >> $GITHUB_OUTPUT
+  ###################################################
+  ## website
+  ###################################################
 
+  website_build:
+    name: Build website using Docusaurus
+    runs-on: ${{fromJSON(inputs.runs_on)}}
+    needs: [project_types,versioned_source]
+    if: |
+      needs.project_types.outputs.cloudflare_website == 'true' &&
+      inputs.cloudflare_website != ''
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Docusaurus Theme
+        run: |
+          git clone https://github.com/product-os/docusaurus-theme
+          cd docusaurus-theme
+          chmod +x docusaurusify.sh
+          # Replace params here with variables
+          ./docusaurusify.sh ${{ github.event.repository.name }} ${{ github.repository_owner }} ${{ github.event.repository.default_branch }} https://${{ inputs.cloudflare_website }}.pages.dev
```

Comment:
This makes me think it might make more sense for `https://github.com/product-os/docusaurus-theme` to expose a github action that we reuse here like the `balena-io/deploy-to-balena-action` in https://github.com/balena-io/deploy-to-balena-action/blob/master/action.yml since right now there needs to be fairly intimate knowledge of how to run `product-os/docusaurus-theme` built into flowzone vs just calling an action with the right variables

Reply:
@Page- good idea - will convert it and update this PR

Reply:
Done!